src/sidebars/star/star.html

```diff
@@ -1,18 +1,14 @@
-﻿<img class="star" ng-src="assets/png/{{starCtrl.favouriteUrl()}}.png" ng-click="starCtrl.click(); $event.stopPropagation();"
+﻿<img class="star" ng-src="assets/png/{{starCtrl.favouriteUrl()}}.png" ng-click="starCtrl.click($event); $event.stopPropagation();"
      ng-mouseenter="starCtrl.mouseEnter()" ng-mouseleave="starCtrl.mouseLeave()" title="{{starCtrl.tooltip()}}" draggable="false" />
-<div class="confirmation-backdrop " ng-if="starCtrl.confirmationShow" ng-click="starCtrl.hideModal()">
-  <div class="confirmation-modal" ng-click="$event.stopPropagation();">
-    <div class=modal-content>
-      <div class="header">
-        <span class="title">StockFlux</span>
-        <icon name="close" icon-click="starCtrl.hideModal()"></icon>
-      </div>
+<div class="confirmation-backdrop" ng-if="starCtrl.confirmationShow" ng-click="starCtrl.hideModal()">
+  <div class="bubble-head" ng-style="{top:starCtrl.modalBubbleTop()+'px'}"></div>
+  <div class="confirmation-modal" ng-class="{flipped: starCtrl.modalFlip()}"  ng-click="$event.stopPropagation();" ng-style="{top:starCtrl.modalTop()+'px'}">
+    <div class=confirmation-content>
       <div class="message">
-        <p ng-bind="starCtrl.confirmationMessage"></p>
+        <span>Are you sure you wish to remove this stock from your favourites?</span>
       </div>
       <div class="buttons">
-        <button class="btn btn-primary ng-binding" ng-click="starCtrl.deselect(); $event.stopPropagation();">OK</button>
-        <button class="btn btn-default ng-binding" ng-click="starCtrl.hideModal(); $event.stopPropagation();">Cancel</button>
+        <button class="btn btn-primary ng-binding" ng-click="starCtrl.deselect(); $event.stopPropagation();">Confirm</button>
```

Comment:
I see your point here. However it's not really an issue with this PR, considering what's already in dev. I could take it out, but then I would end up more or less just be copying the bootstrap styles into our less.